assets/theme-editor.js

```diff
@@ -0,0 +1,20 @@
+document.addEventListener('shopify:block:select', function(event) {
+  const blockSelectedIsSlide = event.target.classList.contains('slideshow__slide');
```

Comment:
@tyleralsbury Do you think we should have a more specific naming for this file or have this block of code as part of a named function?

I don't want to worry so much about how we'll use this file & function in the future, but `assets/theme-editor.js` and a generic `shopify:block:select` event doesn't immediately tell me this file is purely to update the selected block on slideshow component.

Reply:
You're right about this probably being a little too generically named.  I think we should isolate all the theme editor code into a single file, so I think the one file is a good idea, but the naming could be better. Let's revisit this because I'd wager there are a few spots we could improve the editor experience with this type of code.

I noticed a spot in Sofia's collapsible rows PR where it would help if the row whose text you're editing would open when the section reloads, for example.  I'll create an issue for us to audit editor behaviours and look into expanding the amount of editor-only code we have improving that experience and the refactor can happen then.